.github/workflows/_deploy_cdk.yml

```diff
@@ -174,7 +174,7 @@ jobs:
         uses: metriport/deploy-with-cdk@master
         with:
           cdk_action: "diff"
-          cdk_version: "2.87.0"
+          cdk_version: "2.122.0"
```

Comment:
⚠️ just wanted to flag that bumping CDK versions sometimes breaks things, so a head up incase something in the deploy goes sour that we didn't expect